api_tests/src/image.spec.ts

```diff
@@ -48,6 +50,24 @@ test("Upload image and delete it", async () => {
   const content = downloadFileSync(upload.url);
   expect(content.length).toBeGreaterThan(0);
 
+  // Ensure that it comes back with the list_media endpoint
+  const listMediaRes = await alphaImage.listMedia();
+  expect(listMediaRes.images.length).toBe(1);
+
+  // Ensure that it also comes back with the admin all images
+  const listAllMediaRes = await alphaImage.listAllMedia({
+    limit: imageFetchLimit,
+  });
+
+  // This number comes from all the previous thumbnails fetched in other tests.
+  const previousThumbnails = 30;
```

Comment:
This would be annoying if you want to run only this specific test to debug something. I would rather do listAllMedia before uploading anything, then check that the number increased by one after upload.

Reply:
I agree its annoying, but I tried to add image.spec.ts before all the other tests, and all kinds of other issues popped up. I'll try again.